src/hooks/usePropsGetters/useSelectInputProps.ts

```diff
@@ -112,7 +113,8 @@ export default function useSelectInputProps(
     if (
       (currentQuestionData?.type === QuestionTypes.SingleSelect ||
         currentQuestionData?.type === QuestionTypes.SingleFilterValue) &&
-      singleSelectClicked.current
+      singleSelectClicked.current &&
+      nextQuestionOnSingleSelect
```

Comment:
Could you clarify what's the use case of this additional option? Since it's a single select question it should move forward when a single value has been selected. Why should we give the option of stopping that logic?